src/modules/Common/components/TextContent.module.css

```diff
@@ -27,31 +31,31 @@
     }
   }
 
-  & h3 {
+  & h3:not(:global(.mb__0)) {
```

Comment:
> That's quite a complex selector, wouldn't it be worth adding a specific class in this case? 🙂

Reply:
I don't think so: `hx` have styles inherent in the fact that they are in `<TextContent>` component  and when we want to delete specically the margin of a style we add the class `mb__0`.

But I agree with you, the selector is a bit complex.

Reply:
My usual approach in such a case is to specify the properties for all the targeted elements, and to override the values in the specific class. That's how CSS is really good at.

In this case, I would just add at the end of these declarations something like:

```
& .mb__0 {
  margin-bottom: 0;
}
```

Thanks to selector specificity, the class selector would override all the tag selectors 🙂

Reply:
Done with https://github.com/ambanum/OpenTermsArchive.org/pull/169/commits/e4240e8a07fb3404fbcf3bd06cb6e6539a8d33c9